xclim/indices/_threshold.py

```diff
@@ -1584,3 +1722,40 @@ def _exceedance_date(grp):
     out = c.clip(0).resample(time=freq).map(_exceedance_date)
     out.attrs["units"] = ""
     return out
+
+
+@declare_units(snd="[length]", thresh="[length]")
+def winter_storm(snd: xarray.DataArray, thresh: str = "25 cm", freq="AS-JUL"):
+    """
+    Days with snowfall over threshold.
+
+    Number of days with snowfall accumulation greater or equal to threshold.
+
+    Parameters
+    ----------
+    snd : xarray.DataArray
+      Surface snow depth.
+    thresh : str
+      Threshold on snowfall accumulation require to label an event a `winter storm`.
+    freq : str
+      Resampling frequency.
+
+    Returns
+    -------
+    xarray.DataArray
+      Number of days per period identified as winter storms.
+
+    Notes
+    -----
+    Snowfall accumulation is estimated by the change in snow depth.
+    """
+    thresh = convert_units_to(thresh, snd)
+
+    # Compute daily accumulation
+    acc = snd.diff(dim="time")
```

Comment:
Same thing here..  Does a winter storm necessarily derive from what is accumulated on the ground? I think we should be able to calculate a snowfall event from precipitation data as well

Reply:
We had discussions around that with @aulemahal. It could be another indice function, or just additional optional input variables, but that comes with a risk of having the same indicator identifier with different computing approaches.